firmware/.vscode/launch.json

```diff
@@ -0,0 +1,34 @@
+// AUTOMATICALLY GENERATED FILE. PLEASE DO NOT MODIFY IT MANUALLY
```

Comment:
This file is full of paths specific to your computer. We should either gitignore it (if it can be automatically re-generated by platformio) or find a way so it doesn't use absolute paths.

Can you test what happens if you close the project, delete this file, then re-open it